controller/controller.go

```diff
@@ -69,6 +69,9 @@ const (
 	// DefaultMetricsPort is the default port to expose the metrics endpoint
 	DefaultMetricsPort = 8090
 
+	// DefaultPProfPort is the default port to expose the pprof endpoint
+	DefaultPProfPort = 6060
```

Comment:
I recognize that `DefaultMetricsPort` and `DefaultHealthzPort` are being used in a more complex setup for those servers in their respective files. I wanted to mimic that pattern as close as possible without clogging up `main.go`. Open to suggestions here.